### PR TITLE
Website: Leaderboard phase 2 scaffolding 

### PR DESCRIPTION
**Pull request by michellewong793**
This PR begins implementing the leaderboard phase 2 by adding the scaffolding 
image

### CODE REVIEW

**Comment by bkase** [comment]
Is there a page we can see as a preview?

**Comment by figitaki** [comment]
@bkase https://deploy-preview-5006--o1website2.netlify.app/leaderboard

**Comment by michellewong793** [comment]
> Is there a page we can see as a preview?

included screenshot too

File: frontend/website/src/pages/LeaderboardPage.re [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+                  </a>
+                </Next.Link>
+                <span className=Theme.Link.basic>
+                  <Svg
+                    link="/static/img/Icon.Info.svg"
+                    className=Styles.icon
+                    dims=(1.0, 1.0)
+                    alt="a undercase letter i inside a blue circle"
+                  />
+                  {React.string("Last manually updated May 1 2020")}
```
Comment (figitaki) [inline frontend/website/src/pages/LeaderboardPage.re:209]: Can we interpolate a a formatted date here? Even if we're not pulling it from the backend just so it's easier to plug into once we have the data setup.

Comment (michellewong793) [reply frontend/website/src/pages/LeaderboardPage.re:209]: what do you mean interpolate? like "May" + "1" + "2020"?

File: frontend/website/src/pages/LeaderboardPage.re [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+        flexDirection(`column),
+        justifyContent(`center),
+      ]);
+  };
+  [@react.component]
+  let make = () => {
+    <div className=Styles.flexRow>
+      <div className=Styles.flexColumn>
+        <h2 className=Styles.statistic> {React.string("Participants")} </h2>
+        <p className=Styles.value> {React.string("456")} </p>
```
[the remark this answers is not in the document]
Comment (michellewong793) [reply frontend/website/src/pages/LeaderboardPage.re]: got it

**Pull request merged**